src/storage/storage.cpp

```diff
@@ -287,12 +201,12 @@ int Storage::Run()
 
     // load timestamp size
     std::string m_timestamp;
-    if (boost::filesystem::exists(timestamp_path))
+    if (boost::filesystem::exists(config.timestamp_path))
```

Comment:
Should deriving suffixes from base path _and_ file existence be checked in the config's ctor?